br/docs/en/2021-12-09-TiDB-log-based-incremental-backup-draft-1.md

```diff
@@ -0,0 +1,459 @@
+## Structure
```

Comment:
shouldn't this be placed in `/docs/design/`?

Reply:
The structure of this document seems doesn't match the template (lacking of test design, alternative, etc..).

(However, the template doesn't be applied for many docs... Maybe just replace `Structure` with `Introduction` , add a TOC and replace `Design` with `Detailed Design` would be fine?)